morf-core/src/main/java/org/alfasoftware/morf/sql/element/Criterion.java

```diff
@@ -554,23 +554,11 @@ public String toString() {
    */
   @Override
   public void drive(ObjectTreeTraverser traverser) {
-    traverser.dispatch(getField());
-    visitValue(traverser,getValue());
     traverser
-      .dispatch(getSelectStatement())
-      .dispatch(getCriteria());
-  }
-
-
-  @SuppressWarnings("rawtypes")
-  private void visitValue(ObjectTreeTraverser traverser,Object value) {
-    if (value instanceof AliasedField) {
-      traverser.dispatch( (AliasedField)value);
-    } else if (value instanceof Iterable) {

```

Comment:
We've lost this Iterable handling. Not required any more?

Reply:
It's now handled by the traverser (always should have been).  `visitValue(transfers, getValue())` has become `traverser.dispatch(value)`.  That handles and traverses through iterables automatically.